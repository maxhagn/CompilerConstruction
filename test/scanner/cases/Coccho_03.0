    3 5end      end7

    yo mama