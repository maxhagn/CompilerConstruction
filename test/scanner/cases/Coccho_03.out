num 3
num 5
end
id end7
id yo
id mama